test/integration/generated/lro/src/tracing.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+import { createTracingClient } from "@azure/core-tracing";
+
+export const tracingClient = createTracingClient({
+  namespace: "Microsoft.Autorest.LRO",
+  packageName: "Azure.Autorest.LRO"
```

Comment:
Same here, no package version.

Actually, I don't recall if these samples have a dummy SDK_VERSION constant, is that why the property is not being generated?

Reply:
Nah, we should have the package version name. I'll add that, thanks for keeping me honest :P